cmake/ThirdParty.cmake

```diff
@@ -60,21 +60,31 @@ if (OPENASSETIO_ENABLE_PYTHON)
 
     # Get path to Python executable in the created venv, and a way to activate
     # the venv itself.
-    if (WIN32)
-        set(OPENASSETIO_PYTHON_VENV_ACTIVATE "${CMAKE_INSTALL_PREFIX}/Scripts/activate.bat")
-        set(OPENASSETIO_PYTHON_VENV_EXE "${CMAKE_INSTALL_PREFIX}/Scripts/python.exe")
+
+    if (OPENASSETIO_ENABLE_PYTHON_TEST_VENV)
+        if (WIN32)
+            set(OPENASSETIO_PYTHON_VENV_ACTIVATE "${CMAKE_INSTALL_PREFIX}/Scripts/activate.bat")
+            set(OPENASSETIO_PYTHON_VENV_EXE "${CMAKE_INSTALL_PREFIX}/Scripts/python.exe")
```

Comment:
[consider] Should this var still have `VENV` in the name now?

Reply:
Yeah I guess we can strip the `VENV` now, so the variable name works for both branches.

Reply:
Would probably help with readability at the call site too as it becomes clear that the python isn't always in a venv?

Reply:
Removed VENV from name in 9a1951d9dbd56dacecdb245888a7534c81d7f2b1